server/recceiver/cfstore.py

```diff
@@ -502,20 +507,22 @@ def __updateCF__(proc, pvInfoByName, delrec, hostName, iocName, iocid, owner, io
     if proc.cancelled:
         raise defer.CancelledError()
 
-def create_properties(owner, iocTime, hostName, iocName, iocid):
+def create_properties(owner, iocTime, recceiver_id, hostName, iocName, iocid):
```

Comment:
I think, we should use the same convention here between `receiver_id` and `iocid`. Please select it either convention to keep the similar style, even if we have multiple convention. I don't want to see more conventions in a code.

Reply:
I think that's good advice; there are already two conventions in that line (`hostName` vs. `iocid`), adding a third is probably overkill :smile: